apis/events.d.ts

```diff
@@ -69,7 +69,7 @@ export class Request<T = any> extends Event<T> {
 
   subject: ref
 
-  reply (results: any): void
+  reply (results: any, /** @beta */ options?: object): void
```

Comment:
Ah, now I understand! You want to offer an additional parameter to an existing method, and passing that parameter is beta.
That would be solved by providing an overload:

```suggestion
  reply (results: any): void
  /** @beta */
  reply (results: any, options: object): void
```
Note that I removed the `?:`, as the optionality is achieved by having one overload without and one with the `options` parameter.

can we be more specific about the type of `options`?